### ISSUE TITLE
minor installation process updates

### ISSUE THREAD

**Issue by sofroniewn** [labsyspharm/minerva-story#63]
hello! I'm one of the JOSS reviewers for your submission here https://github.com/openjournals/joss-reviews/issues/2579, just getting going with things.

I was able to install successfully, but might suggest adding a few extra clarifying lines where I had to "guess" what to do.

First was I ran into a permissions error on `gem install jekyll bundler`. I used `sudo gem install jekyll bundler` and it worked fine, but am not sure if that is recommended. A comment to that effect might help a beginner.

Before running `bundle install` I had to clone the repository and navigate to the main directory. While this may be obvious in retrospect a line or two explaining that could also help a beginner (the next instruction contains the helpful advice `In the project folder, run command`)

**Comment by jmuhlich** [comment]
Hi Nick, thanks for the feedback. We can definitely add these to the docs. Since the initial manuscript submission we've been working on removing Jekyll and thus Ruby/Gem/Bundler from the workflow entirely, instead adding live preview support directly to Minerva Author. Not sure it's quite ready to replace the current Jekyll build process and revise the manuscript though. We can discuss in our daily meeting tomorrow.

**Comment by thejohnhoffer** [comment]
Hi @sofroniewn,

We've since updated Minerva Story to no longer require Ruby or Jekyll. I've also updated the README.md of this repository to be explicit about cloning the repository and changing to the required directory. We've included installation instructions for installing `npm` and `npx` to allow the users to run a simple `http-server`, though any simple HTTP server would work just as well.

We'll soon update the manuscript to reflect these changes.
